static/js/stream_settings_ui.js

```diff
@@ -558,21 +557,21 @@ export function setup_page(callback) {
         const sort_toggler = components.toggle({
             values: [
                 {
-                    label_html: `<i class="fa fa-sort-alpha-asc tippy-bottom tippy-zulip-tooltip" data-tippy-content="${$t(
-                        {defaultMessage: "Sort by name"},
-                    )}"></i>`,
+                    label_html: `<i class="fa fa-sort-alpha-asc" data-tippy-content="${$t({
+                        defaultMessage: "Sort by name",
+                    })}" aria-hidden="true"></i>`,
```

Comment:
Why are you adding `aria-hidden`?

Reply:
Ahh, this came from the previous approach i was planning to do this change but that is not required here in any sense;
Updated the PR;